app-admin/salt/salt-3003.1.ebuild

```diff
@@ -143,6 +143,9 @@ python_prepare() {
 	local abc
 	abc="$("${EPYTHON}" -c 'import collections.abc; print("|".join((c for c in dir(collections.abc) if not c.startswith("_"))))')" || die
 	find -name '*.py' -type f -print0 | xargs -0 sed -r -e "s:collections\\.(${abc}):collections.abc.\\1:g" -i || die
+
+	# removes contextvars, see bug: https://bugs.gentoo.org/799431
+	sed -i '/^contextvars/d' requirements/base.txt || die
```

Comment:
Is doing this enough? I see a few `import contextvars` in the output of `grep contextvars $(qlist app-admin/salt)`.

Reply:
Indeed, it does seem to be enough. Tested locally.